app/views/users/passwords/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Change your password</h2>
```

Comment:
Yes, your understanding is correct :)

And thanks for the tips on those HTML patterns, I'll keep then in mind from now on! 

And `@minimum_password_length` is a variable that Devise provide us like a view helper. It's value is defined on `config/initializers/devise.rb`:

```
config.password_length = ArchiveConfig.PASSWORD_LENGTH_MIN..ArchiveConfig.PASSWORD_LENGTH_MAX